### ISSUE TITLE
Improve the the Query Log

### ISSUE THREAD

**Issue by ameshkov** [AdguardTeam/AdguardDNS#319]
1. Add "Client" column: https://uploads.adguard.com/up04_rihff_AdGuard_DNS_.png
2. Mark requests which were blocked by AdGuard DNS (with a red background). Print why it was blocked in the "Response" cell.

**Comment by hmage** [comment]
@IldarKamalov @maximtop `/querylog` field -- `client`:

```json
{
    "client": "127.0.0.1",
    "elapsed_ms": "0.276011",
    "question": {
        "class": "IN",
        "host": "doubleclick.net",
        "type": "A"
    },
    "reason": "FilteredBlackList",
    "rule": "||doubleclick.net^",
    "status": "NXDOMAIN",
    "time": "2018-08-31T19:58:24+03:00"
}
```

**Comment by hmage** [comment]
@IldarKamalov @maximtop  If `reason` begins with `"Filtered"`, show it in red. On hover, popup with contents from `rule`.

**Comment by hmage** [comment]
Fixed in e62050fb7ee1c38fdaa5ce7b06ae553cf0a9060f
